apps/dataset-ingestion/app/app.sh

```diff
@@ -17,8 +17,12 @@ export INCLUDE_TRAIN
 LOAD_CELEBAHQ=${LOAD_CELEBAHQ:false}
 export LOAD_CELEBAHQ
 
+
 if [ -z "${WF_DATA_SOURCE_GCP_BUCKET}" ]; then
     echo "Please set the WF_DATA_SOURCE_GCP_BUCKET environment variable"
+    python3 status.py --completed 0 \
```

Comment:
this `status.py` is a different path than the other ones?

can't tell if it's a changing-path kind of thing, but perhaps it would be better to make a constant reference to it and use that everywhere?